corehq/apps/motech/openmrs/urls.py

```diff
@@ -0,0 +1,23 @@
+from django.conf.urls import url
+from corehq.apps.motech.openmrs import views
+
+urlpatterns = [
+    # concepts
+    url(r'^rest/concept/$',
+        views.all_openmrs_concepts,
+        name='all_openmrs_concepts'),
+    url(r'^rest/concept/search/$',
+        views.concept_search,
+        name='openmrs_concept_search'),
```

Comment:
It's not actually called here yet, but it's setting up an endpoint that pages can use in an async call. Tempted to just leave it here, since I'm planning to use it soon.